app/src/main/java/com/denisatrif/truthdare/viewmodel/PlayersViewModel.kt

```diff
@@ -1,36 +1,34 @@
 package com.denisatrif.truthdare.viewmodel
 
+import android.app.Application
 import androidx.compose.runtime.mutableStateListOf
 import androidx.lifecycle.LiveData
 import androidx.lifecycle.MutableLiveData
 import androidx.lifecycle.ViewModel
 import androidx.lifecycle.viewModelScope
+import com.denisatrif.truthdare.R
 import com.denisatrif.truthdare.db.model.Player
 import com.denisatrif.truthdare.db.repos.PlayersRepository
 import dagger.hilt.android.lifecycle.HiltViewModel
 import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.flow.Flow
 import kotlinx.coroutines.launch
 import javax.inject.Inject
 
 @HiltViewModel
-class PlayersViewModel @Inject constructor(private val playersRepository: PlayersRepository) : ViewModel() {
+class PlayersViewModel @Inject constructor(
+    private val playersRepository: PlayersRepository,
+    private val app: Application
```

Comment:
Using Context, Activity or Application in ViewModel is a big NO,  since it causes memory leak. 
As far as I know, it is only used to write in shared prefs. That can be done in the ComposePlayersScreen, directly there, or with an util, or an extension function